gammapy/datasets/map.py

```diff
@@ -1725,6 +1732,131 @@ def to_image(self, name=None):
+        plot_kwargs = kwargs.copy()
+        plot_kwargs.update(kwargs_counts)
+        plot_kwargs.setdefault("label", "Counts")
+        plot_kwargs.setdefault("add_cbar", True)
+        plot_kwargs.setdefault("cmap", "coolwarm")
+
+        ax = self.counts.sum_over_axes().plot(ax=ax, **plot_kwargs)
+
+        return ax
+
+    def plot_masks(self, ax=None, kwargs_fit=None, kwargs_safe=None):
+        """Plot mask safe and mask fit
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes`
+            Axes to plot on.
+        kwargs_fit: dict
+            Keyword arguments passed to `~RegionNDMap.plot_mask()` for mask fit.
+        kwargs_safe: dict
+            Keyword arguments passed to `~RegionNDMap.plot_mask()` for maks safe
+
+        Returns
+        -------
+        ax : `~matplotlib.axes.Axes`
+            Axes object.
+        """
+        from matplotlib import colors as colors
+        kwargs_fit = kwargs_fit or {}
+        kwargs_safe = kwargs_safe or {}
+
+        c_white = colors.colorConverter.to_rgba('white', alpha=0)
+        c_black = colors.colorConverter.to_rgba('black', alpha=1)
+        cmap_rb = colors.LinearSegmentedColormap.from_list('rb_cmap', [c_black, c_white], 512)
+
+        kwargs_fit.setdefault("label", "Mask fit")
+        kwargs_fit.setdefault("cmap", cmap_rb)
+        kwargs_safe.setdefault("label", "Mask safe")
+        kwargs_safe.setdefault("cmap", cmap_rb)
+
+        if self.mask_fit:
+            self.mask_fit_image.plot(ax=ax, **kwargs_fit)
+
+        if self.mask_safe:
+            self.mask_safe_image.plot(ax=ax, **kwargs_safe)
+
+        return ax
+
+    def plot_excess(self, ax=None, kwargs_excess=None, **kwargs):
+        """Plot excess and predicted signal.
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes`
+            Axes to plot on.
+        kwargs_excess: dict
+            Keyword arguments passed to `~matplotlib.axes.Axes.errorbar` for
+            the excess.
+        **kwargs: dict
+            Keyword arguments passed to both plot methods.
+
+        Returns
+        -------
+        ax : `~matplotlib.axes.Axes`
+            Axes object.
+        """
+        kwargs_excess = kwargs_excess or {}
+
+        plot_kwargs = kwargs.copy()
+        plot_kwargs.update(kwargs_excess)
+        plot_kwargs.setdefault("label", "Excess counts")
+        plot_kwargs.setdefault("add_cbar", True)
+        plot_kwargs.setdefault("cmap", "coolwarm")
+        ax = self.excess.sum_over_axes().plot(ax, **plot_kwargs)
+
+        return ax
+
+    def peek(self, fig=None):
+        """Quick-look summary plots.
+
+        Parameters
+        ----------
+        fig : `~matplotlib.figure.Figure`
+            Figure to add AxesSubplot on.
+
+        Returns
+        -------
+        ax1, ax2, ax3 : `~matplotlib.axes.AxesSubplot`
+            Counts, excess and exposure.
+        """
+        fig = get_figure(fig, 16, 4)
+        ax1, ax2, ax3 = fig.subplots(1, 3)
```

Comment:
Creating the subplots this way misses the WCS info for the plot. You have to pass the `subplot_kw`, see e.g. https://github.com/gammapy/gammapy/blob/master/gammapy/maps/core.py#L1028